modules/balana-core/src/test/java/org/wso2/balana/advance/AdvanceTestV3.java

```diff
@@ -131,26 +140,76 @@ public void testAdvanceTest0003() throws Exception {
         }
     }
 
+
+    public void testAdvanceTest0004() throws Exception {
+
+        String reqResNo;
+        Set<String> policies = new HashSet<String>();
+        policies.add("TestPolicy_0004.xml");
+        log.info("Advance Test 0004 is started. This test is for Jira https://github.com/wso2/balana/issues/73");
+
+        for (int i = 1; i < 2; i++) {
```

Comment:
We can remove this for loop and straightaway pass the  complete filename to `TestUtil.createRequest`